dashboard/test/fixtures/schools.tsv

```diff
@@ -0,0 +1,17 @@
+NCESSCH	LEAID	SCHNAM	CHARTR
+010000200277	0100002	SEQUOYAH SCH - CHALKVILLE CAMPUS	N
+010000201402	0100002	EUFAULA SCH - EUFAULA CAMPUS	N
+010000500870	0100005	ALA AVENUE MIDDLE SCH	N
+010000500871	0100005	ALBERTVILLE HIGH SCH	N
+020000100206	0200001	JOANN A. ALEXIE MEMORIAL SCHOOL	2
+020000100207	0200001	GLADYS JUNG ELEMENTARY	2
+020000300216	0200003	ALAKANUK SCHOOL	2
+020000300217	0200003	EMMONAK SCHOOL	2
+060000113121	0600001	ASSURANCE LEARNING ACADEMY	1
+060000113717	0600001	ALBERT EINSTEIN ACADEMY ELEMENTARY	1
+060001411118	0600014	SUMMIT LEADERSHIP ACADEMY-HIGH DESERT	1
+060001411746	0600014	PATHWAYS TO COLLEGE	1
+480000407173	4800004	POR VIDA ACADEMY CHARTER H S	1
+480000409116	4800004	CORPUS CHRISTI COLLEGE PREP H S	1
+480001407174	4800014	GEORGE GERVIN ACADEMY	1
+480001410415	4800014	THE EDUCATION AND TRAINING CENTER	1
```

Comment:
nit: It doesn't really matter what this data is, but it might be nice if the subset includes Seattle public schools (for no reason other than availability heuristic based on our location, for demos, dev testing, etc)